admin_tests/pre_reg/test_views.py

```diff
@@ -25,7 +25,8 @@
 
 
 class TestDraftListView(AdminTestCase):
-    def setUp(self):
+    @mock.patch('admin.pre_reg.views.DraftRegistration.get_metadata_files')
```

Comment:
Why does this need to be mocked? It's not making any outgoing calls, and you're not assigning any side effects / return values / etc. to it here.